pkg/images/defaults/flavor.go

```diff
@@ -231,7 +235,8 @@ func OpenSourceImageFlavor() ImageFlavor {
 		ScannerDBSlimImageName: "scanner-db-slim",
 
 		ChartRepo: ChartRepo{
-			URL: "https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/",
+			URL:     "https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/",
+			IconURL: "https://github.com/stackrox/stackrox/blob/master/image/templates/helm/shared/assets/StackRox_icon.png",
```

Comment:
There are already:
1. https://github.com/stackrox/stackrox/blob/master/image/templates/helm/stackrox-central/assets/icon.png
2. https://github.com/stackrox/stackrox/blob/master/image/templates/helm/stackrox-secured-cluster/assets/icon.png

I suggest that you make those to be symlinks to this file `helm/shared/assets/StackRox_icon.png` to avoid having three copies of the same ting.
But please check that the `Application` resource deployed on the cluster looks sane. I mean this one https://github.com/stackrox/stackrox/blob/master/image/templates/helm/stackrox-central/templates/00-stackrox-application.yaml

Reply:
I have replaced these with softlinks, as Git wouldn't recognize hardlinks. Checking the Application resource next

Reply:
Correction: Softlinks break the build process for roxctl, as they won't be included.
This might explain why we had the same icon in both assets folders already. Moved back to hardlinks.
```
make image
mkdir -p bin/{darwin,linux,windows}
RACE=0 CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 (...)/go/src/github.com/stackrox/stackrox/scripts/go-build.sh ./roxctl
Compiling Go source in ./roxctl to bin/darwin/roxctl
image/embed_charts.go:25:33: pattern templates/helm/stackrox-central/*: cannot embed directory templates/helm/stackrox-central/assets: contains no embeddable files
make: *** [cli-build] Error 1
```

Reply:
Looks like you don't even need links. Files from `shared/` directory are present in each chart.
I ran this:
```sh
make cli
roxctl helm output central-services --output-dir blah-central
roxctl helm output secured-cluster-services --output-dir blah-secured-cluster
```

And got this:
![image](https://user-images.githubusercontent.com/537715/179232897-c19ffa7f-c8e6-4a5d-956c-e561d9b78d41.png)

Reply:
That was a great observation, thank you!
I've updated the Helm application CRD to conditionally include the right icon, based on the default image registry and have removed the duplicated image from the two chart folders.